### PR TITLE
Added deastiny layout for contra [based on dvorak type II]

### PR DESCRIPTION
**Pull request by DeastinY**

### CODE REVIEW

File: keyboards/contra/keymaps/deastiny/config.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/contra/keymaps/deastiny/config.h]: Could you replace the boiler plate: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "config_common.h"
```
with just `#pragma once`, and remove the endif at the bottom?

File: keyboards/contra/keymaps/deastiny/keymap.c [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+ | ESC  |   Ü  |   ,  |   .  |   P  |   Y  |   F  |   G  |   C  |   T  |   Z  | Bksp |
+ |------+------+------+------+------+-------------+------+------+------+------+------|
+ | TAB  |   A  |   O  |   E  |   I  |   U  |   H  |   D  |   R  |   N  |   S  |  L   |
+ |------+------+------+------+------+------|------+------+------+------+------+------|
+ | Shift|   Ä  |   Ö  |   Q  |   J  |   K  |   X  |   B  |   M  |   W  |   V  |Enter |
+ |------+------+------+------+------+------+------+------+------+------+------+------|  
+ | CTRL |  GUI | Alt  | MOD  |Lower |   Space     |Raise | Left |Right |  Up  |Down  |
+ `-----------------------------------------------------------------------------------'
+**/
+  [_DVORAK] = LAYOUT_planck_mit(
```
Comment (drashna) [inline keyboards/contra/keymaps/deastiny/keymap.c]: This should be `LAYOUT_ortho_4x12`

File: keyboards/contra/keymaps/deastiny/config.h (hunk 1 above)
Comment (DeastinY) [reply keyboards/contra/keymaps/deastiny/config.h]: so just delete all of it and only write the pragma part?

Comment (drashna) [reply keyboards/contra/keymaps/deastiny/config.h]: Yes, delete those three lines (and the one at the end), and replace it with `#pragma once`.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**